rules_java_gapic/java_gapic_pkg.bzl

```diff
@@ -13,14 +13,27 @@
 # limitations under the License.
 
 load("@com_google_api_gax_java_properties//:dependencies.properties.bzl", "PROPERTIES")
+load("@com_google_protobuf//:protobuf_version.bzl", "PROTOBUF_JAVA_VERSION")
+
+def _syncVersionWithGoogleapis(properties):
+    syncedProterties = {}
+    for dep, version in properties.items():
```

Comment:
I don't think you need to iterate items to set a value to a key. 


properties[k] = v

Reply:
done